src/libm/sleefsimdsp.c

```diff
@@ -22,7 +24,11 @@ extern const float rempitabsp[];
 
 #ifdef ENABLE_SSE2
 #define CONFIG 2
+#if !defined(SLEEF_GENHEADER)
 #include "helpersse2.h"
+#else
+#include "macroonlySSE2.h"
+#endif
```

Comment:
Do we need to header files? Couldn't we  conditionally compile the same header file, exposing only the macros when SLEEF_GENHEADER is defined? This could avoid the need of generating the `macroonly*` files.

Reply:
Instead of only exposing some macros, I chose the approach to define some of the macros beginning with //@.

Below is comparison between pros and cons.
Approach 1: Exposing a part of the macros according to whether SLEEF_GENHEADER is defined.
Pros: The mechanism is straightforward.
Cons: We need to protect most of the part in the macro with large #ifdef. This is hard to read.
Generation of macros is not very understandable anyway.

Approach 2: Define some of the macros beginning with //@
Pros: Easier to read and manage the source code.
Cons: This is a dedicated way for SLEEF. and requires understanding how the macros are generated.